flow/designs/sky130hd/chameleon_hier/macro_placment.cfg

```diff
@@ -0,0 +1,8 @@
+
+
+_3294_ R0 279.68 456.96
+_3293_ R0 279.68 3067.48
+_3292_ R0 2404.42 3067.48
+ahb_sys_0_uut.S0.CACHE R0 2404.42 2007.36
+ahb_sys_0_uut.apb_sys_inst_0 R0 4530.08 3106.92
+ibex_core R0  4530.08 4317.32
```

Comment:
why doesn't tritonmacroplace perform this?

Reply:
Yes, tritonMP can peform this, this custom macro config was done to force macro pin shapes to be aligned to top-level routing tracks and therefore assist pin access. But this is unnecessary and have removed this now.